### PR TITLE
[WIP] ML.NET landing page

### PR DESCRIPTION
**Pull request by briacht**
New landing page

https://review.docs.microsoft.com/en-us/dotnet/machine-learning/mldotnet-index?branch=pr-en-us-11626

[AB#1485103](https://dev.azure.com/mseng/677da0fb-b067-4f77-b89b-f32c12bb8617/_workitems/edit/1485103)

### CODE REVIEW

File: docs/machine-learning/welcome-to-mldotnet.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+---
```
Comment (mairaw) [inline docs/machine-learning/welcome-to-mldotnet.md]: nit: i'd name the file just welcome.md

File: docs/machine-learning/welcome-to-mldotnet.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+title: Welcome to ML.NET
+description: Learn about ML.NET, a free, open-source, and cross-platform machine learning framework that enables you to build custom AI solutions and integrate them into your .NET applications.
+author: briacht
+ms.custom: seodec18
```
Comment (mairaw) [inline docs/machine-learning/welcome-to-mldotnet.md]:
```suggestion
```

**Review by mairaw — COMMENTED** [review]
Not sure if you're working on this or not, but left you some quick comments

**Comment by natke** [comment]
Closing this, as it has been superseded by #13712

**Pull request closed without merging**